tests/baselines/reference/user/TypeScript-Node-Starter.log

```diff
@@ -0,0 +1,17 @@
+Exit Code: 2
+Standard output:
+node_modules/connect-mongo/src/types.d.ts(113,66): error TS2694: Namespace 'Express' has no exported member 'SessionData'.
```

Comment:
connect-mongo updated recently. I don't think TypeScript-Node-Starter is not valuable anymore, so I'll probably remove it.